picard/formats/id3.py

```diff
@@ -177,6 +180,18 @@ class ID3File(File):
     __rtranslate_freetext = dict([(v, k) for k, v in __translate_freetext.items()])
     __translate_freetext['writer'] = 'writer'  # For backward compatibility of case
 
+    # Freetext fields that are loaded case-insensitive
+    __rtranslate_freetext_ci = {
```

Comment:
Isn't this redundant with https://github.com/metabrainz/picard/pull/1263/files#diff-0d55c0ba51fcdcf1cab8dfe3acac98b0R166 ?

Reply:
Not really, one is ASF the other ID3. So while in this specific case it is the same list of tags this is just coincidence and quickly can be different when we add other tags to the list. E.g. we could decide to also read `MusicBrainz Artist Id` case insensitive for ID3 (because this has been discussed before). And the mapping is different for ASF.

What we could look into is to unify the way we specify the mappings here. E.g. ASF by default always has the mapping specified as "picard name => format name", while ID3 and MP4 use "format name => picard name". The first was easier to work with in this case. Would probably be easier if this would be the same across all formats, at least when possible. Also the naming of the mapping constants could be unified (`__TRANS` vs. `__translate_freetext` vs. `__freeform_tags` vs `__translate`).

Reply:
Thanks for clarification. Keep as is for now.